frontend/src/app/_components/creator-page/creator-page.component.html

```diff
@@ -0,0 +1,21 @@
+<app-breadcrumb [lastCrumbText]="creator.name"></app-breadcrumb>
+<div class="row creator-page">
+    <div class="col-sm-4 col-6">
+        <div class="row">
```

Comment:
I wouldn't bother with row and col-12 in here and just set your margins and such here.

Reply:
but then i have to think lol.